src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private LocalDateTime orderDate;
+    private Long userId;
+
```

Comment:
Need we return userId if frontender ask the order by the id (they know this id)?